bigtable/metrics.go

```diff
@@ -66,11 +67,20 @@ const (
 	metricUnitCount = "1"
 )
 
-// These are effectively const, but for testing purposes they are mutable
+// These are effectively constant, but for testing purposes they are mutable
 var (
 	// duration between two metric exports
 	defaultSamplePeriod = 5 * time.Minute
 
+	// Logger used to log errors seen while collecting and exporting metrics
+	// Log format is yyyy/MM/dd HH:mm:ss bigtable: .........
+	// Date and time are in local timezone
+	//
+	// Errors seen while collecting and exporting metrics should not fail user operation.
+	// So, not all errors are bubbled up to the user.
+	// These error logs will help in debugging any metrics related issues.
+	metricsLogger = log.New(os.Stderr, "bigtable: ", log.LstdFlags|log.Lmsgprefix)
```

Comment:
nit: If this is only used for logging errors, could we call it `metricsErrorLogger`?